src/presentation/http/router/note.test.ts

```diff
@@ -49,49 +50,68 @@ describe('Note API', () => {
   });
 
   describe('GET note/:notePublicId ', () => {
-    test('Returns note with access rights by public id with 200 status when note is publicly available', async () => {
+    test.each([
+      { role: MemberRole.Read,
+        isPublic: false,
+        isAuthorized: true,
+        statusCode: 200 },
+
+      { role: MemberRole.Write,
+        isPublic: false,
+        isAuthorized: true,
+        statusCode: 200 },
+
+      { role: null,
+        isPublic: false,
+        isAuthorized: true,
+        statusCode: 403 },
+
+      { role: null,
+        isPublic: false,
+        isAuthorized: false,
+        statusCode: 403 },
+
+      { role: null,
+        isPublic: false,
+        isAuthorized: false,
+        statusCode: 403 },
+    ])
+    ('Returns note with access rights by public id. Request is done by user who is anon, not in team or in team with different rights', async ({ role, isPublic, isAuthorized, statusCode }) => {
```

Comment:
Put the information from the second sentence in the comments of the test cases inside test.each